kiota_abstractions/response_handler.py

```diff
@@ -1,28 +0,0 @@
-from abc import ABC, abstractmethod
-from typing import Callable, Dict, Optional, TypeVar
-
-from .serialization import Parsable, ParsableFactory
-
-NativeResponseType = TypeVar("NativeResponseType")
-ModelType = TypeVar("ModelType")
-
-
-class ResponseHandler(ABC):
```

Comment:
don't you want to keep this interface so you can use it with the new request option introduced in the http package?

Reply:
I agree.  Makes sense to keep it to use during instantiation of a response handler option.

Has been retained.